CHANGELOG.md

```diff
@@ -21,6 +21,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - New gpio driver for STM32 with nif and port support for read and write functions.
 - Added support for interrupts to STM32 GPIO port driver.
 - Added suppoprt for PicoW extra gpio pins (led) to the gpio driver.
+- Added support for setting the receive buffer size for sockets
```

Comment:
This actually goes under the umbrella "Added support for the OTP socket interface" on line 18.

Reply:
I re-worded to to make it a little more explicit.  If you don't think it requires a call-out, we can drop the line, for sure.